terraform/clusters/dougneal.gds-re-run-sandbox.aws.ext.govsvc.uk/kubeconfig

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Config
+clusters:
+- name: dougneal-cluster
+  cluster:
+    server: https://dougneal.run-sandbox.aws.ext.govsvc.uk:6443
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURGekNDQWYrZ0F3SUJBZ0lSQUlmMjlCaHMzWEZZVnhlQ2loM1ZFRVl3RFFZSktvWklodmNOQVFFTEJRQXcKSlRFUk1BOEdBMVVFQ2hNSVltOXZkR3QxWW1VeEVEQU9CZ05WQkFNVEIydDFZbVV0WTJFd0hoY05NVGd4TVRJdwpNVEl5TmpReVdoY05NVGt4TVRJd01USXlOalF5V2pBbE1SRXdEd1lEVlFRS0V3aGliMjkwYTNWaVpURVFNQTRHCkExVUVBeE1IYTNWaVpTMWpZVENDQVNJd0RRWUpLb1pJaHZjTkFRRUJCUUFEZ2dFUEFEQ0NBUW9DZ2dFQkFPaTMKeVU1N0RBWVp0RUcxYkp3TWtQYUpER2lYc3BkeUIxU3pPQ1l0TUFSMzlrUWhGaWJLUVFZYUIxOUpibEZqNGJCNwoxQkRFajBIRG83N3NkNUZLem44ZzQ1QTI4VjJRcVVkV1RSL1dkRGJjb20wZjZobWt6UlVEWmF1UkxUSVJJMlhxCkFSZVViMzdvb1FDM2FJbHZVUnNYNitHODgzeHNQdmRPVDViajVKMFdkL0REMVZjNHV1KzgyenNGZ3BmQ0liWXcKRTh1WkttQWYyYzY4elZGb1dSQi9xRDRsYm02bmVBMldqZm9vTFo3U2ovd3pVbGw5WTNvaVRSVmdkZjZFY2NaTApZMU1pM2N5THoyZFJEQWpLQWNRQWpzeEUwS1BscFZsTE12M2VMSmxtRzRUSGhIUHVOU1lpcytOaktsUDNhNDhEClR3TkhxYWNPNjRDYXE1ZjNIdGtDQXdFQUFhTkNNRUF3RGdZRFZSMFBBUUgvQkFRREFnS2tNQThHQTFVZEV3RUIKL3dRRk1BTUJBZjh3SFFZRFZSME9CQllFRkR0MS80cVNLcUI1SEw0b0RBQm5UM0RRL01MNU1BMEdDU3FHU0liMwpEUUVCQ3dVQUE0SUJBUUM5cUZuRlNXVzNzYVVpSENEZjA3YnlaUnZaTFVDMi9FSWdtSVRTcFdORE0vbWdtTGhNClVYNm9QL1VBd2xLRk95cDIzVUd1UUg2K1MxS1VoWXhHMUwvZ3NlUkNyN1dlNzFjS0k0MStoeXIzQ3JkZGhwZmcKbDdiTGhmdWVkRDJKTG9HUGhqYjhhdGFMOEtvenhjM3RuQ1ZBS0JOdzRXMExXSkYybkNTTVc3MTJUOHdzMHg0awpIWmptdGl0OHNwRWQrcjBPQWpWdEVabGlZMnZEQTN1b0RJZk5ISUEyRnBUM1FwYi9QYk1sV1diVEhYQXhKMFRkCi9TcTdscjZ1d0ZzZ2szVTZiWHFHSVc1eUdFTzRIeEJXTWxpY3YrOVRvTyt2MkJ4RFBYTDFyMnZIUFo1ZmVnS0cKZnJ6WWxBbGUvcnB2RktRRmdDVEZDdW16cEZ1dVRCTVpLSzZ1Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
```

Comment:
I don't think this corresponds to the current generation of your cluster. I get a "untrusted, unknown CA" error when trying to use this kubeconfig.